### PR TITLE
fix/connection Generation WalletId access Fallback

### PR DESCRIPTION
**Pull request by alazarevski**
Fix walletId not accessible from State case.

### CODE REVIEW

File: src/actions/oldInvitationsActions.js [hunk 1]
```diff
@@ -142,7 +142,9 @@ export const fetchOldInviteNotificationsAction = () => {
     });
 
     await dispatch(updateConnectionsAction());
-    dispatch(getExistingChatsAction());
+    await dispatch(getExistingChatsAction());
```
Comment (poocart) [inline src/actions/oldInvitationsActions.js]: `return dispatch(getExistingChatsAction())` will await for promise to complete

File: src/actions/oldInvitationsActions.js [hunk 2]
```diff
@@ -142,7 +142,9 @@ export const fetchOldInviteNotificationsAction = () => {
...
+
+    return Promise.resolve(true);
```
Comment (jsidorenko) [inline src/actions/oldInvitationsActions.js]: why do we need this? (just curious)

Comment (alazarevski) [reply src/actions/oldInvitationsActions.js]: during debug, it ran async even though await was called before the dispatch

File: src/actions/oldInvitationsActions.js (hunk 1 above)
Comment (alazarevski) [reply src/actions/oldInvitationsActions.js]: Changed and tested in 2a6fa98

**Pull request merged**